blueprints/networking/vpc-connectivity-lab/variables.tf

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "ip_ranges" {
+  description = "Subnet/Routes IP CIDR ranges."
+  type        = map(string)
+  default = {
+    ext         = "10.255.0.0/16"
+    hub-a       = "10.0.1.0/24"
+    hub-all     = "10.0.0.0/16"
+    hub-b       = "10.0.2.0/24"
+    hub-nva     = "10.0.0.0/24"
+    int         = "10.0.0.0/9"
+    peering-a   = "10.3.0.0/24"
+    peering-b   = "10.4.0.0/24"
+    rfc1918_10  = "10.0.0.0/8"
```

Comment:
nit: I would hardwire these

Reply:
That's where I came from, but other blueprints use the same data structure, and I don't dislike it.